bin/supported

```diff
@@ -24,22 +25,17 @@ const { writeToCsv } = require('../lib/output/csv-output');
     }
 
     if (cli.flags.json && result) {
-      const { projectName } = result;
-      delete result.projectName;
       console.log(
         JSON.stringify(
-          { ...result, project: { name: projectName, path: projectPath, type: 'node_module' } },
+          projectPaths.length > 1 ? result : result.projects[0],
```

Comment:
should we output a different JSON structure when emitting 1 project then when emitting 2? Or would it be simpler to simply emit the same structure for both cases?

Reply:
Updated the PR with that change. Whatever you said make sense, if we supply different JSON depending on a number of input, the consumer will have a tough time adding if clause.